### PR TITLE
Child Disability Allowance - Social Security Act

### LINKED ISSUE #25: Child Disability Allowance

### PR DESCRIPTION
**Pull request by Br3nda**
* Tax and benefit system evolution: Child Disability Allowance.

- - - -

These changes add new features:
* Child Disability allowance from the Social Security Act (closes #25)

### CODE REVIEW

File: openfisca_aotearoa/variables/social_security/child_disability_allowance.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            resident_or_citizen * \
+            is_principal_carer * \
+            has_disabled_child
+
+
+class social_security__family_has_disabled_child(Variable):
+    value_type = bool
+    entity = Family
+    definition_period = MONTH
+    label = u'Does the family have a child who meets the criteria for disabled'
```
Comment (timClicks) [inline openfisca_aotearoa/variables/social_security/child_disability_allowance.py]: Unicode literals are probably not required?

Comment (Br3nda) [reply openfisca_aotearoa/variables/social_security/child_disability_allowance.py]: We've gone with adding them before the strings that appear to the human users. It's a convention we learned from France's implementation.

**Comment by timClicks** [comment]
Sorry about that! I didn't take the time to read the project's guidelines,
e.g. https://openfisca.org/doc/coding-the-legislation/10_basic_example.html,
for adding the comment

**Comment by Br3nda** [comment]
I had a chat with the Parliamentary Council's Office - and they are keen on the idea that we DRY the open fisca rules -- but tag it someway to we can find it and attempt to DRY the legislation behind it too.

**Pull request merged**